src/components/Navbar.js

```diff
@@ -0,0 +1,32 @@
+const Navbar = () => {
+    return (
+        <nav className="navbar d-flex flex-justify-evenly flex-align-center">
+            <div className="nav-section-left logo d-flex flex-justify-evenly">
+                <ion-icon name="beer-outline"></ion-icon>
+                wine & spirits
+            </div>
+            <div className="nav-section-center d-flex flex-justify-evenly">
+                <p className="d-flex flex-justify-evenly flex-align-center">Shop
+                    <ion-icon name="caret-down-outline"></ion-icon>
+                </p>
+                <p className="d-flex flex-justify-evenly flex-align-center">Our wines
+                    <ion-icon name="caret-down-outline"></ion-icon>
+                </p>
+                <p>About us</p>
+                <p>Blog</p>
```

Comment:
I think for semantic HTML, it would be better to have them as buttons rather than p tag.